src/ReactSelectMe.js

```diff
@@ -238,6 +238,12 @@ export default class ReactSelectMe extends PureComponent {
         onFocus={this.onSearch}
         onClick={this.onSearch}
         onPaste={this.onSearch}
+        onKeyUp={(e) => {
+          /* Internet Explorer detection. See more
+           * http://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser
+           */
+          if (false || !!document.documentMode) this.onSearch(e);
```

Comment:
I had some research about `false` in IE checking condition.
Special comment `/*@cc_on!@*/`, which prepends it, is a fuature of IE 6-10. So this `false` is unneccessary for IE11 checking.